validation_layers/build.gradle

```diff
@@ -0,0 +1,151 @@
+plugins {
+    id 'com.android.library'
+    id 'org.jetbrains.kotlin.android'
+}
+
+def khronosOpenxrSourceDir = file("../thirdparty/openxr-source")
+def khronosOpenxrValidationLayersAndroidLibraryTargetDir = file("${buildDir}/temp/openxr_validation_layers")
+def khronosOpenxrValidationLayersAndroidAssetsTargetDir = file("${buildDir}/temp/openxr_validation_layers_assets")
+def khronosOpenxrValidationLayersAndroidBuildDir = file("${buildDir}/temp/openxr_validation_layers_build")
+
+android {
+    compileSdk versions.compileSdk
+    ndkVersion versions.ndkVersion
+
+    defaultConfig {
+        minSdk versions.minSdk
+        targetSdk versions.targetSdk
+        versionName getReleaseVersion()
+
+        setProperty("archivesBaseName", "openxr-validation-layers")
+
+        ndk {
+            //noinspection ChromeOsAbiSupport
+            abiFilters "arm64-v8a", "x86_64"
+        }
+    }
+
+    namespace = "org.godotengine.openxr.validation_layers"
+
+    sourceSets {
+        main {
+            jniLibs.srcDirs += [
+                "${khronosOpenxrValidationLayersAndroidLibraryTargetDir}/arm64-v8a",
+                "${khronosOpenxrValidationLayersAndroidLibraryTargetDir}/x86_64",
+            ]
+
+            // We don't need to give separate values for arm64-v8a and x86_64 - just one of them is fine.
+            assets.srcDirs += ["${khronosOpenxrValidationLayersAndroidAssetsTargetDir}/arm64-v8a"]
+        }
+    }
+
+    packagingOptions {
+        if (shouldNotStrip()) {
+            doNotStrip '**/*.so'
+        }
+    }
+
+    compileOptions {
+        sourceCompatibility versions.javaVersion
+        targetCompatibility versions.javaVersion
+    }
+
+    kotlinOptions {
+        jvmTarget = versions.javaVersion
+    }
+}
+
+dependencies {
+    implementation "androidx.fragment:fragment:$versions.fragmentVersion"
```

Comment:
I don't think we need this since we have no java / kotlin code.

Reply:
Removed!